code/components/jomjol_flowcontroll/ClassFlowCNNGeneral.cpp

```diff
@@ -95,8 +95,15 @@ string ClassFlowCNNGeneral::getReadout(int _analog = 0, bool _extendedResolution
                 else {
                     prev = PointerEvalHybridNew(GENERAL[_analog]->ROI[GENERAL[_analog]->ROI.size() - 1]->result_float, prev, prev);
                 }
-                result = std::to_string(prev);
-                LogFile.WriteToFile(ESP_LOG_DEBUG, TAG, "getReadout(dig100)  prev=" + std::to_string(prev));
+
+                // is necessary because a number greater than 9.994999 returns a 10! (for further details see lines 159 to 161)
```

Comment:
```suggestion
                // is necessary because a number greater than 9.994999 returns a 10! (for further details see check in PointerEvalHybridNew)
```

We should not use line number references as they can change quickly